platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/state/DefaultStateManagementComponent.java

```diff
@@ -35,11 +35,7 @@
 import com.swirlds.logging.payloads.InsufficientSignaturesPayload;
 import com.swirlds.platform.components.common.output.FatalErrorConsumer;
 import com.swirlds.platform.components.common.query.PrioritySystemTransactionSubmitter;
-import com.swirlds.platform.components.state.output.IssConsumer;
-import com.swirlds.platform.components.state.output.NewLatestCompleteStateConsumer;
-import com.swirlds.platform.components.state.output.StateHasEnoughSignaturesConsumer;
-import com.swirlds.platform.components.state.output.StateLacksSignaturesConsumer;
-import com.swirlds.platform.components.state.output.StateToDiskAttemptConsumer;
+import com.swirlds.platform.components.state.output.*;
```

Comment:
We explicitly avoid importing using wildcards. It makes for verbose lists of imports, but the fear is that overuse of wildcards will lead to everything importing everything else.

Reply:
Reformatting did this automatically... does yours not?

Reply:
Fixed